readme.md

```diff
@@ -333,17 +333,16 @@ end
 
 With this action, we want to create an instance using the params the user entered into the form.
 
-**Note** When the request completes, we now need to worry about where to direct the user after adding a new artist.
+**Note:** When the request completes, we now need to worry about where to direct the user after adding a new artist.
 
-### Render vs Redirect (5 / 110)
+### Render vs Redirect (5 min)
 
-Rails has variety of ways to map our app's logic navigating a user's request to a response they care about.
+Rails has variety of ways to map our applications's logic navigating a user's request to a response they care about.
 
 The one way you will use the most often is a `redirect`. Using Rails `redirect_to` helper method, we can take a request from a client, and then without sending a response back right away, we can make a new request back to the browser, which will trigger the response we want.
 
```

Comment:
Is this the first time we talk about helper methods? think it might be. maybe mention what those are.